From 88d98f468850c0cae91ce6262d11616b87c941bd Mon Sep 17 00:00:00 2001
From: Pepe Iborra <pepeiborra@gmail.com>
Date: Mon, 6 Nov 2017 20:17:20 +0000
Subject: [PATCH] Add Cabal Setup.hs script

---
 Setup.hs | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 Setup.hs

diff --git a/Setup.hs b/Setup.hs
new file mode 100644
index 0000000..9a994af
--- /dev/null
+++ b/Setup.hs
@@ -0,0 +1,2 @@
+import Distribution.Simple
+main = defaultMain
-- 
2.13.6

